packages/xsnap-lockdown/src/paths.js

```diff
@@ -5,6 +5,7 @@ export const bundlePaths = {
   lockdown: new URL('../dist/lockdown.bundle', import.meta.url).pathname,
   lockdownDebug: new URL('../dist/lockdown-debug.bundle', import.meta.url)
     .pathname,
+  objectInspect: new URL('../dist/object-inspect.js', import.meta.url).pathname,
```

Comment:
Should we align this file with #10082?
```js
import { fileURLToPath } from 'url';

const resolveRelative = relativePath =>
  fileURLToPath(new URL(relativePath, import.meta.url));
```
```suggestion
  lockdown: resolveRelative('../dist/lockdown.bundle'),
  lockdownDebug: resolveRelative('../dist/lockdown-debug.bundle'),
  objectInspect: resolveRelative('../dist/object-inspect.js'),
```

Reply:
May’s well snip the sword down and stow it in the Locker of Damocles.